src/tribler-gui/tribler_gui/tribler_window.py

```diff
@@ -437,10 +437,16 @@ def tray_show_message(self, title, message):
             except RuntimeError as e:
                 logging.error("Failed to set tray message: %s", str(e))
 
-    def on_tribler_started(self, version):
+    def on_tribler_started(self, version, public_key):
         if self.tribler_started:
             logging.warning("Received duplicate Tribler Core started event")
             return
+
+        if public_key:
+            # if public key format will be changed, don't forget to change it
+            # at the core side as well
+            SentryReporter.set_user(public_key.encode('utf-8'))
```

Comment:
:+1:

Reply:
Similarly here, using `mid` could be privacy preserving